pkgs/net-test/tests/net/git-checkout.rkt

```diff
@@ -59,13 +59,13 @@
    [else
     (error 'compare "no such file: ~s" a)]))
 
-(when git-exe
+(when (and git-exe (not (getenv "GITHUB_ACTIONS")))
```

Comment:
Why change this here?

Reply:
The version of `git` that comes with the GH actions Linux environment is too old and doesn't support the `-b` flag to `git init`. We could try to update it in the action, but this seemed easier (and this test is already not being run).